Remotion/Web/Test.NetFramework/Global.asax.resx

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
I think we can drop this one instead of moving it over to the new project. Please try/test.